packages/bemlinter/src/snapshot.js

```diff
@@ -0,0 +1,93 @@
+const _ = require('lodash');
+const fs = require('fs');
+
+function getLogIdentifiers(log) {
+  return _.pick(log, ['moduleName', 'blockName', 'message']);
+}
+
+function getActualQuality(lintResult) {
+  return {
+    errorList: lintResult.getErrorList().map(getLogIdentifiers),
+    warningList: lintResult.getErrorList().map(getLogIdentifiers),
+  };
+}
+
+function getSnapshotQuality(filePath) {
+  if (!fs.existsSync(filePath)) {
+    return false;
+  }
+
+  const data = fs.readFileSync(filePath).toString();
+  try {
+    return JSON.parse(data);
+  } catch(error) {
+    return false;
+  }
+}
+
+function getSnapshotResult(lintResult, snapshotQuality) {
+  const completeLogList = (logList, logType) => {
+    const isSnapshotExists = snapshotQuality !== false;
+    const isModuleExists = log => _.some(snapshotQuality[`${logType}List`], _.pick(log, ['moduleName', 'blockName']));
+    const isErrorExists = log => _.some(snapshotQuality[`${logType}List`], getLogIdentifiers(log));
+    const completeLog = log => {
+      log.isNew = isSnapshotExists && isModuleExists(log) && !isErrorExists(log);
+
```

Comment:
Yes, I don't add an empty line when the `return` is in a one-line block :thought_balloon: